index.js

```diff
@@ -321,4 +372,56 @@ input DeleteOtherAnswerInput {
   parentAnswerId: ID!
 }
 
+input CreateRoutingRuleSetInput {
+  questionPageId: ID!
+}
```

Comment:
Could this take a object of the next level as well, same as the CreateRoutingRuleInput?

Reply:
Possibly, yeah, I think it depends how we end up using the API. This might be something that we revisit when we integrate the API with the front-end. I think it depends if the ruleset gets created before/at the same time as the rules.